tests/php/WordPressExternalConnectionTest.php

```diff
@@ -129,7 +130,6 @@ public function test_push() {
 			'post_excerpt' => 'post excerpt',
 			'post_name'    => 'slug',
 			'post_status'  => 'publish',
-			'post_type'    => $post_type,
```

Comment:
Why was this removed?

Reply:
This is a duplicate entry in the array. Please check the whole array value for more information.